Makefile

```diff
@@ -1120,7 +1126,7 @@ ifdef LANGUAGES
 endif
 	$(BINDIST_CMD)
 
-export ODIR _OBJS LDFLAGS CXX W32FLAGS DEFINES CXXFLAGS TARGETSYSTEM CLANG PCH PCHFLAGS
+export ODIR _OBJS LDFLAGS CXX W32FLAGS DEFINES CXXFLAGS TARGETSYSTEM CLANG PCH PCHFLAGS PCH_SUFFIX
```

Comment:
That line has been there since 2013: ec16dc4822f4c34a4b04c160e706eca04c6e3880. You can see that `_OBJS` was used in the tests Makefile back then, but that export is clearly out of date.

Reply:
Yeah, thought it might be something like that, but I'd rather checked before cleaning it out.